atomicapp/cli/main.py

```diff
@@ -188,12 +209,12 @@ def set_arguments(self):
 
         parser_install.add_argument(
             "--destination",
-            dest="target_path",
+            dest="destination",
             default=None,
             help="Destination directory for install")
 
         parser_install.add_argument(
-            "APP",
+            "app_spec",
```

Comment:
@rtnpro @charliedrage let me know if you can come up with a better variable name than `app_spec`